README.md

```diff
@@ -31,6 +32,11 @@ cd gke-logging
 poetry install
 ```
 
+To get full functionality from the ASGI module install with:
+```
+poetry install -E asgi
+```
+
```

Comment:
Thanks! I recommend adding similar instructions to the recommended install method as well.